app/components/ab_test/cookies_banner_component/cookies_banner_component.html+gds.slim

```diff
@@ -0,0 +1,14 @@
+.govuk-cookie-banner.cookies-banner-component--gds role="region" aria-label = t("cookies_preferences.banner.aria.label")
+  .govuk-cookie-banner__message.govuk-width-container
+    .govuk-grid-row
+      .govuk-grid-column-two-thirds
+        h2.govuk-cookie-banner__heading.govuk-heading-m = "Cookies on Teaching Vacancies"
```

Comment:
havent added translations, i think bit premature as this is a test

Reply:
In which case you could get rid of outputting a static string:

```slim
h2.govuk-cookie-banner__heading.govuk-heading-m Cookies on Teaching Vacancies
```